program/src/instruction.rs

```diff
@@ -1,15 +0,0 @@
-use borsh::{BorshDeserialize, BorshSerialize};
-
-#[derive(BorshSerialize, BorshDeserialize, PartialEq, Debug)]
-pub enum EclipseInstruction {
```

Comment:
I dropped this because I couldn't see any need due to being the only instruction and its deserialization causing extra compute costs for the on-chain program.